guides/working-with-examples.md

```diff
@@ -0,0 +1,47 @@
+# Working with Examples
+
+Adding meaningful examples can greatly enrich the documentation of components in the Cashmere library. Here are some things to remember when creating or modifying examples.
+
+## Examples files
+
+The following files are used in the creation of examples for the Cashmere documentation website.
+
+### Files you may need to modify
+
+-   `projects/cashmere-examples/src/lib/{EXAMPLE_NAME}` - the folder containing all files for an example
```

Comment:
nitpicky, but its usually called directory, not folder

Reply:
noted